README.md

```diff
@@ -14,47 +14,56 @@ substrate-api-client a library written in Rust for connecting to the substrate's
 ## Prerequisites
 
 In order to build the substrate-api-client and the examples, Rust and the wasm target are needed. For Linux:
-
-    curl https://sh.rustup.rs -sSf | sh
-
-    rustup default nightly
-    rustup target add wasm32-unknown-unknown --toolchain nightly
-
+```
+curl https://sh.rustup.rs -sSf | sh
+rustup show
```

Comment:
Maybe a comment that `rustup show` automatically installs the suitable toolchain would be nice.

Reply:
Better now?